### DISCUSSION TITLE
How do I find the correct Pimcore documentation version

### DISCUSSION THREAD

**Discussion by RobertoGongora** [pimcore/pimcore#17393] [Q&A / Support]
I'm using Pimcore 11.1.5 - I'm trying to locate a specific module documentation but it's extremely difficult to pinpoint which documentation "Version" I need to review.

There are mentions to features not available on my Pimcore UI in some latest versions, so I don't feel at all confident about the docs I'm reading since they may contain either too old or too new info, I'm just not sure.

The versioning mention in the docs is a bit... strange. How do I know that Pimcore 11.0.x belongs to `2023.1` and what does `2023.2` and `2024.1` include? How do I know which versions match each other? Why is this so confusing and not just use the Pimcore versions themselves?

https://pimcore.com/docs/platform/Platform_Version/#versioning-schema-and-release-cycles

How am I supposed to know this:

> For example: within 2023.1 it can install pimcore/pimcore:11.0.1 and pimcore/pimcore:11.0.2 as soon as it is released, but not pimcore/pimcore:11.1.0.

**Comment by fashxp** [comment]
Hi,
we recommend to use the platform-version in your projects ... then it fits also to the documentation and you also can leverage the other [benefits](https://pimcore.com/docs/platform/Platform_Version/#benefits-in-a-nutshell). 

If for what ever reason this is not possible, you can either check [these tables](https://pimcore.com/docs/platform/Platform_Version/Platform_Version_Releases/2024.1) for the versions, or you just use the documentation directly in the corresponding github repository, e.g. for [here the core-framework](https://github.com/pimcore/pimcore/blob/v11.0.1/doc/README.md).
